pkg/webui/console/components/device-import-form/index.js

```diff
@@ -45,12 +50,17 @@ const m = defineMessages({
   advancedSectionTitle: 'Advanced end device claiming settings',
   infoText:
     'You can use the import functionality to register multiple end devices at once by uploading a file containing the registration information in one of the available formats. For more information, see also our documentation on <DocLink>Importing End Devices</DocLink>.',
+  fallbackValuesImport:
+    'These values will be used in case your file does not provide them. They are not required, although if not provided here or in the imported file, the import of the end device will not be successful',
```

Comment:
```suggestion
    'These values will be used in case the imported file does not provide them. They are not required, although if not provided here or in the imported file, the import of the end device will not be successful.',
```
Use end of sentence periods when the notification consists of multiple sentences: https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.18/DESIGN.md#general-considerations